Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6141,6 +6141,21 @@ private MediaAttachment GetMediaAttachment(IReadOnlyList<ResultSetValue> reader)
 
 #nullable enable
 
+        private static int GetCacheSizeOrDefault()
+        {
+            // https://www.sqlite.org/pragma.html#pragma_cache_size
+            // Positive values == pages to cache (by default, each pasge is 4KiB).
+            // Negative values == calculate number of pages to cache under this number (in KiB).
+
+            if (Environment.GetEnvironmentVariable("JELLYFIN_SQLITE_CACHE_SIZE") is { } environmentVariable
```

Comment:
We use `using Microsoft.Extensions.Configuration;` instead if getting the env var directly.
You can set the default value in `Emby.Server.Implementations//ConfigurationOptions.cs`

Reply:
So set this in the `ctor` instead of within this static context?